src/status_im/contexts/wallet/send/transaction_confirmation/view.cljs

```diff
@@ -153,7 +153,14 @@
           (reduce-kv (fn [acc k v]
                        (assoc acc k {:amount v :token-symbol token-symbol}))
                      {}
-                     network-amounts)]
+                     network-amounts)
+          network-values-sanitized (into {}
+                                         (map (fn [[k v]]
+                                                [k
+                                                 (if (money/equal-to (v :amount) 0)
+                                                   (assoc v :amount "<0.00")
```

Comment:
What does `<0.00` mean? Did you mean `<0.01` ?

Reply:
Do we have this code else where? Maybe it's better to reuse those methods then

Reply:
Maybe it should be `~0.00`. But not sure if this is needed. Currently there's a bug in the input amount screen that allows to introduce more decimals than the token decimals count. That leads `money/with-precision` function to count it as 0 if for example a token has 6 decimals and user enters `0.0000009` (more than 6 decimals), causing elements not being shown in `quo/summary-tag` component. If we fix that in input amount, I think this should not be needed.

Reply:
Maybe it should be `~0.01` (or `<0.01`). It's good that we initiate a discussion with the Desktop team about this and align Mobile with Desktop.

> If we fix that in input amount, I think this should not be needed.

We should create a separate issue to fix that separately, IMO. Since we deal with money, it's good that we add these checks wherever possible to prevent any surprises.